webview/src/plots/components/App.test.tsx

```diff
@@ -192,29 +212,27 @@ describe('App', () => {
     })
   })
 
-  it('should render only checkpoint plots when given a message with only checkpoint plots data', () => {
+  it('should render other sections given a message with only checkpoint plots data', () => {
     renderAppWithOptionalData({
       checkpoint: checkpointPlotsFixture
     })
 
     expect(screen.queryByText('Loading Plots...')).not.toBeInTheDocument()
     expect(screen.getByText('Trends')).toBeInTheDocument()
-    expect(screen.queryByText('Data Series')).not.toBeInTheDocument()
-    expect(screen.queryByText('Images')).not.toBeInTheDocument()
+    expect(screen.getByText('Data Series')).toBeInTheDocument()
+    expect(screen.getByText('Images')).toBeInTheDocument()
+    expect(screen.getByText('No Plots to Display')).toBeInTheDocument()
+    expect(screen.getByText('No Images to Compare')).toBeInTheDocument()
   })
 
-  it('should render checkpoint and template plots when given messages with both types of plots data', () => {
+  it('should render checkpoint even when there is no checkpoint plots data', () => {
     renderAppWithOptionalData({
-      checkpoint: checkpointPlotsFixture
-    })
-
-    sendSetDataMessage({
       template: templatePlotsFixture
     })
 
     expect(screen.queryByText('Loading Plots...')).not.toBeInTheDocument()
     expect(screen.getByText('Trends')).toBeInTheDocument()
-    expect(screen.getByText('Data Series')).toBeInTheDocument()
+    expect(screen.getByText('No Metrics Selected')).toBeInTheDocument()
```

Comment:
[Q] Can we hide the context menu dropdown(s) when there are no actual plots for the project that we are showing? I think that at the moment the "filter metrics" dropdown would show as empty?

Reply:
Done